### PR TITLE
Homework3

### PR DESCRIPTION
**Pull request by egor200201**

### CODE REVIEW

File: task3/avarage/avarege.go [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package main
+
+import "fmt"
+
+func main() {
+	a := [6]int{1, 2, 3, 4, 5, 6}
+	fmt.Println(Average(a))
+}
+
+func Average(a [6]int) float64 {
```
Comment (nikonok) [inline task3/avarage/avarege.go]: Please rename file, because it name contains error (compare with function name)

File: task3/mapbykeys/map.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+func printSorted(m map[int]string) {
+
```
Comment (nikonok) [inline task3/mapbykeys/map.go]: Remove new line from start at all functions.

File: task3/reverse/reverse.go [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package main
+
+import "fmt"
+
+func main() {
```
Comment (nikonok) [inline task3/reverse/reverse.go]: Move it to function according to HW task instructions.

**Review by nikonok — CHANGES_REQUESTED** [review]
Please, add some more information output to your tasks. Not only "result".
And here also `package main.go` file in the branch, idk why it not shown at the diff. Maybe you need to update branch before merging
image

Maybe that will help you https://stackoverflow.com/questions/23748973/is-it-neccessary-to-update-git-branch-before-merging-it-to-develop-master

**Review by denislituev — APPROVED** [review]

**Review by nikonok — APPROVED** [review]

**Pull request merged**